inform7/extensions/basic_inform/Sections/Glulx and Glk.w

```diff
@@ -0,0 +1,149 @@
+Glulx and Glk.
+
+Support for Glulx and Glk interpreter systems.
+
+@ =
+Part Four - Glulx and Glk (for Glulx only)
+
+@h Glk windows.
+Minimal support for Glk windows. Other extensions may extend the kind.
+
+=
+Chapter - Glk windows
+
+A glk window is a kind of abstract object.
+The glk window kind is accessible to Inter as "K_Glk_Window".
+The specification of a glk window is "Models the Glk window system."
+
+A glk window has a number called the rock number.
+The rock number property translates into Inter as "glk_rock".
+
+A glk window has a number called the reference number.
+The reference number property translates into Inter as "glk_ref".
+
+@ We create one glk window now, which is used for when there's a Glk event on an
+unknown window.
+
+=
+The unknown window is a glk window.
+The unknown window object is accessible to Inter as "Unknown_Glk_Window".
+
+@h Feature testing.
+These phrases let us test for various interpreter features.
+While most features can use the generic functions, a few need special handling,
+and so individual phrases are defined for them.
+
+=
+Chapter - Glk and Glulx feature testing
+
+To decide whether (F - glk feature) is/are supported:
+	(- glk_gestalt({F}) -).
```

Comment:
For the glk/glulx features that are just binaries, I've defined them as a kind of value, letting them just be evaluated through this one phrase. For the gestalts that would return a number or require a second argument, I've defined separate phrases below.